azext_connect/custom.py

```diff
@@ -11,17 +11,20 @@
 
 SERVICE_MAP = {
     'signalr': ('signalr', 'Azure SignalR Service', 1),
-    'webapp': ('webapp', 'Azure Web App Service', 2)
+    'sql': ('sql', 'Azure SQL Database', 2),
+    'webapp': ('webapp', 'Azure Web App Service', 3),
+    'acr': ('acr', 'Azure Container Registry', 4),
+    'aks': ('aks', 'Azure Kubernetes Service', 5)
 }
 DEFAULT_CLI = get_default_cli()
 
 
-def connect(service, resource_group=None):
+def connect(service, resource_group=None, app_name = None, server = None, sql = None, username = None, password = None, aks_name = None, acr_name = None):
```

Comment:
I think we should consider an object to contain those parameters, otherwise the number of parameters will grow as we are adding more and more services.

Reply:
And the same for the below functions.